packages/template-retail-react-app/app/pages/checkout/partials/payment.jsx

```diff
@@ -84,7 +85,7 @@ const Payment = () => {
             paymentMethodId: 'CREDIT_CARD',
             paymentCard: {
                 holder: formValue.holder,
-                issueNumber: formValue.number.replace(/ /g, ''),
+                maskedNumber: getMaskCreditCardNumber(formValue.number),
```

Comment:
@adamraya Re: whether this is a breaking change or not:
- From a template-extensibility view, it is not a breaking change. `onPaymentSubmit` function is not exported, so it's safe to make the change in this case.
- From a functionality view, yes, this behaviour is different when compared to V2. But this change is for V3, so it's already ok to have a different behaviour here.

Reply:
@vmarta What was the behavior in v2? Does v2 behavior also need to be changed?

Reply:
The issue doesn't exist in v2.